services/QuillLMS/app/controllers/cms/districts_controller.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+class Cms::DistrictsController < Cms::CmsController
+
+  before_action :text_search_inputs, only: [:index, :search]
+  before_action :set_district, only: [
+    :show
+  ]
+
+  DISTRICTS_PER_PAGE = 30
+
+  def index
+    @district_search_query = {}
+    @district_search_query_results = []
+    @number_of_pages = 0
+  end
+
+  def search
+    district_search_query = district_query_params
+    district_search_query_results = district_query(district_query_params)
+    district_search_query_results ||= []
+    number_of_pages = (district_search_query_results.count / DISTRICTS_PER_PAGE).ceil
```

Comment:
You need a `to_f` here. Integer/Integer division in Ruby counts the number of whole items (it's confusing, but you usually want to avoid dividing an integer by an integer).
```suggestion
    number_of_pages = (district_search_query_results.count / DISTRICTS_PER_PAGE.to_f).ceil

```
<img width="385" alt="Screen Shot 2022-04-14 at 4 47 05 PM" src="https://user-images.githubusercontent.com/1304933/163472620-0604f8dd-4fc0-45b7-801c-e731db6a224e.png">